test/lit.cfg

```diff
@@ -1020,16 +1020,16 @@ elif (run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'w
     config.target_runtime = "native"
     config.target_swift_autolink_extract = inferSwiftBinary("swift-autolink-extract")
 
-    libdispatch_artifact_dir = make_path(config.libdispatch_build_path, 'src')
+    libdispatch_artifact_dir = config.libdispatch_build_path
+    libdispatch_swift_module_dir = make_path(libdispatch_artifact_dir, 'src', 'swift', 'swift')
```

Comment:
This path seems…worse? Why the double nesting?

Reply:
I have no idea.  I just built under docker/Linux and looked where things ended up.
See my comment above: https://github.com/apple/swift/pull/27940#issuecomment-547635227

Reply:
I think it's worth fixing the path where things get built rather than adjusting the test logic to look there.

Reply:
The "double" is because the swift overlay is in src/swift, and the swift modules/docs/interfaces are emitted into a swift directory.